__tests__/helpers/wait.ts

```diff
@@ -0,0 +1,5 @@
+import { act } from "@testing-library/react-native"
+
+export const waitForNextRender = async () => {
+  await new Promise((resolve) => setTimeout(resolve, 0))
+}
```

Comment:
you dont have to use async await because you are not using a returned value from an internal promise. Just use

```js
export const waitForNextRender = () => {
  return new Promise((resolve) => setTimeout(resolve, 0))
}
```